koku/api/tags/queries.py

```diff
@@ -378,6 +417,17 @@ def execute_query(self):
         if self.parameters.get("key_only"):
             tag_data = self.get_tag_keys()
             query_data = sorted(tag_data, reverse=self.order_direction == "desc")
+        elif self.parameters.get_filter("value"):
+            tag_data = self.get_tag_values()
+            # This is sorted by values that start with the filter first, then values that contain the filter
+            # based on a discussion with UX
```

Comment:
👏🏻 👏🏻 👏🏻 
I was gonna ask you to put a comment in here since this logic is so weird